pkg/serviceimport/map.go

```diff
@@ -70,6 +70,11 @@ func (m *Map) GetIPs(namespace, name string, checkCluster func(string) bool) ([]
 		return nil, false
 	}
 
+	if cluster != "" {
+		ips, found := clusterIPs[cluster]
+		return ips, found
```

Comment:
```suggestion
		return clusterIPs[cluster]
```

Reply:
This gives `Not enough arguments to return`